src/proto/Polkadot.proto

```diff
@@ -3,24 +3,80 @@ syntax = "proto3";
 package TW.Polkadot.Proto;
 option java_package = "wallet.core.jni.proto";
 
-message ExtrinsicsTransfer {
-    string to_address = 1;
-    bytes value = 2;
+enum Network {
+    POLKADOT = 0;
+    KUSAMA = 2;
+}
+
+enum RewardDestination {
+    STAKED = 0;
+    STASH = 1;
+    CONTROLLER = 2;
+}
+
+message Era {
+    uint64 period = 1;
+    uint64 phase = 2;
```

Comment:
what is `Era`?

Reply:
https://wiki.polkadot.network/docs/en/glossary#era